apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -34,6 +39,104 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
       .then(data => setCourseOfferings(data));
   }, []);
 
+  useEffect(() => {
+    if (!courseOfferings) return;
+    if (!isNewSection) {
+      //  TO DO: refactor for efficiency.  Consider using a flatten-like function (maybe in a helper file?)
+      const highData = {
+        ...courseOfferings[MARKETING_AUDIENCE.HIGH][curriculumTypes.course],
+        ...courseOfferings[MARKETING_AUDIENCE.HIGH][
+          curriculumTypes.standalone_unit
+        ],
+        ...courseOfferings[MARKETING_AUDIENCE.HIGH][curriculumTypes.module],
+      };
+      const middleData = {
+        ...courseOfferings[MARKETING_AUDIENCE.MIDDLE][curriculumTypes.course],
+        ...courseOfferings[MARKETING_AUDIENCE.MIDDLE][
+          curriculumTypes.standalone_unit
+        ],
+        ...courseOfferings[MARKETING_AUDIENCE.MIDDLE][curriculumTypes.module],
+      };
+      const elementaryData = {
+        ...courseOfferings[MARKETING_AUDIENCE.ELEMENTARY][
+          curriculumTypes.course
+        ],
+        ...courseOfferings[MARKETING_AUDIENCE.ELEMENTARY][
+          curriculumTypes.module
+        ],
+      };
+      const hocData = {...courseOfferings[MARKETING_AUDIENCE.HOC]};
```

Comment:
possibly as a follow up, do you also need to expand the PL offerings?

Reply:
Yes - this is on my "next" list.  Right now we wanted to get this out the door so Hannah could work on her part a bit more easily.

Reply:
That's the plan!